clar2wasm/tests/wasm-generation/functions.rs

```diff
@@ -0,0 +1,60 @@
+use clar2wasm::tools::crosscheck;
+use clarity::vm::Value;
+use proptest::proptest;
+use proptest::strategy::Strategy;
+
+use crate::{random_expressions, PropValue, TypePrinter};
+
+const DEFINE_PRIVATE_READONLY: [&str; 2] = ["define-private", "define-read-only"];
```

Comment:
What is the purpose of this constant?

Reply:
How is it different from [this](https://github.com/stacks-network/clarity-wasm/blob/ed807ff4bfe7d7279827c5b1ec02f4a9e79fe141/clar2wasm/tests/wasm-generation/arithmetic.rs#L6)